README.md

```diff
@@ -2091,6 +2091,27 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
     </details>
 
+* <a id='remove-blank-lines-between-chained-functions'></a>(<a href='#remove-blank-lines-between-chained-functions'>link</a>) **Remove blank lines between chained functions.** [![SwiftFormat: blanklinesbetweenchainedfunctions](https://img.shields.io/badge/SwiftFormat-blankLinesBetweenChainedFunctions-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md#blanklinesbetweenchainedfunctions)
```

Comment:
Preview of what the MarkDown looks like when it is rendered: https://gist.github.com/mannylopez/178ea1b36e97f55ede82cda2ed93b7b2

Reply:
It's also easy to preview the markdown using this "Display the rich diff" button on the file in the diff view:

<img width="402" alt="image" src="https://github.com/airbnb/swift/assets/1811727/00231bf5-4f6c-4d4f-9b08-da3f292dff51">

</br></br>

<img width="1014" alt="image" src="https://github.com/airbnb/swift/assets/1811727/0b6d7168-a3ae-4c44-8d38-a9f67d19d14f">

Reply:
TIL! Thanks, Cal!